lib/src/test/java/space/taran/arklib/user/score/ScoreStorageTests.kt

```diff
@@ -0,0 +1,77 @@
+package space.taran.arklib.user.score
+
+import dev.arkbuilders.arklib.ResourceId
+import dev.arkbuilders.arklib.user.score.Score
+import dev.arkbuilders.arklib.user.score.ScoreStorage
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+class ScoreStorageTests {
+
+    private lateinit var testObj: ScoreStorage
+
+    private var setValueCalled: Boolean = false
+    private val randomValue by lazy {
+        (Math.random() * 100).toInt()
+    }
+    private var isRemoveCalled: Boolean = false
+    private val idList: MutableList<ResourceId> = mutableListOf()
+    private lateinit var pair: Pair<ResourceId, Score>
+
+    @Before
+    fun setup() {
+        testObj = object : ScoreStorage {
+            override fun getValue(id: ResourceId): Score {
+                return randomValue
+            }
+
+            override fun setValue(id: ResourceId, value: Score) {
+                setValueCalled = true
+                pair = Pair(id, value)
+            }
+
+            override fun remove(id: ResourceId) {
+                isRemoveCalled = true
+                idList.add(id)
+            }
+
+            override suspend fun persist() {
+            }
+        }
```

Comment:
This doesn't look right.. Looks like `capture` from `mockk` lib.

Reply:
@kirillt the unit tests are meant to verify that GetScore, SetScore, and resetScore methods of the interface ScoreStorage actullay call getValue, setValue, and remove respectively.